src/unity/server/unity_server_control.cpp

```diff
@@ -34,6 +34,10 @@ EXPORT void start_server(const unity_server_options& server_options) {
 
   unity_server_initializer server_initializer;
 
+  if(SERVER) { 
```

Comment:
Rather than stopping and starting the server in this case, would it be better to only start the server if it isn't already started, i.e. change the next six lines to something thing like this:
```
if(!SERVER) { 
    SERVER = new unity_server(server_options);
    SERVER->start(server_initializer);
}

Reply:
No, as the all of the options may change.